chef/cookbooks/neutron/templates/default/linuxbridge_conf.ini.erb

```diff
@@ -20,7 +21,8 @@ tenant_network_type = vlan
 #
 # network_vlan_ranges =
 # Example: network_vlan_ranges = physnet1:1000:2999
-network_vlan_ranges = physnet1:<%= @vlan_start %>:<%= @vlan_end %>
+# NOTE(rhafer): network_vlan_ranges is longer used for ml2. The option is now
```

Comment:
> same here

Fixed in https://github.com/toabctl/barclamp-neutron/commit/fc213eb8071c32aefef5ed4e833db517db9c5f8f